src/style.css

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.star-container {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .star:nth-child(-n+1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .star:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .star:nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
Please combine those selectors by comma `,`. I have mentioned that on previous review